learn/configuration/instance_options.md

```diff
@@ -12,13 +12,15 @@ These startup options affect your entire Meilisearch instance, not just a single
 
 Meilisearch accepts a configuration file in the `.toml` format as an alternative to command-line options and environment variables. Configuration files can be easily shared and versioned, and allow you to define multiple options.
 
-**Environment variables can overwrite the configuration file, and command-line options can overwrite environment variables.**
+**If used simultaneously, environment variables can overwrite the configuration file, and command-line options can overwrite environment variables.**
 
 You can download the configuration file using the following command:
 
 <CodeSamples id="instance_options_config_file_download_1" />
 
-Once downloaded, Meilisearch will use it as the default configuration file. You will see it when you launch Meilisearch:
+This default configuration file is divided into sections separated by comments. Each key has a brief description followed by a link to the relevant section of the documentation. Some keys are commented out as they are not required when launching Meilisearch, for example, `import_snapshot`, or because the value provided in the file is not the default value like `max_indexing_memory`.
```

Comment:
"Some keys are commented out as they are not required when launching Meilisearch" => No key are required when launching Meilisearch.

From my POV this is not really useful to "justify" our default config file, but only my opinion

Reply:
I believe "key" here stands for "option", no? In which case, I recommend saying option to avoid misunderstandings.

In any case, I tend to agree with with @curquiza: not sure this paragraph is necessary since we're only describing the file.